packages/gestalt-datepicker/src/DateRange.css

```diff
@@ -28,3 +28,24 @@ html[dir="rtl"] .borderRight {
   transform: translateY(-50%);
   width: 3px;
 }
+
+/* secondary range styles  */
+
+:global ._gestalt .react-datepicker__day--in-range-secondary {
```

Comment:
any reason these are global? If we support custom classnames, can we just pass the classname

Reply:
you are right, we don't need the global here 👍